CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.7.0 (2022-08-03)
+
+### [#21](https://github.com/ayrat555/fang/pull/21)
+
+- Create a full functional async processing module for fang which uses `tokio` to summon processes that are more lighter than threads.
+- Renaming code Job starting names into Task [#29](https://github.com/ayrat555/fang/pull/29).
+- Exporting serde correctly, here is the [commit](https://github.com/ayrat555/fang/pull/21/commits/cf2ce19c970e93d71e387526e882c67db53cea18).
+- Migrate all blocking module code into async 
```

Comment:
let's remove this ;]

Reply:
Solve in [3754904](https://github.com/ayrat555/fang/pull/59/commits/3754904ae18865fb969d9f736237308d3c0934b2)